packages/core-web/src/styles/markbind.css

```diff
@@ -184,13 +184,13 @@ li.footnote-item:target {
     display: block;
 }
 
-.hljs:not(.inline):not(.no-line-numbers) > span {
+.hljs:not(.inline).line-numbers:not(.no-line-numbers) > span {
```

Comment:
The `:not(.no-line-numbers)` selector can likely be omitted with your changes further below